style.css

```diff
@@ -0,0 +1,1043 @@
+  padding: 1rem;
+}
+
+.mid-article-head-content-1-h1 h1 {
+  margin-left: 4rem;
+  font-size: 1.2rem;
+}
+
+.mid-article-head-content-1 img {
+  width: 4rem;
+  height: 4rem;
+  margin-top: -40px;
+  margin-left: 1rem;
+}
+
+.mid-article-head-content-1-a1 {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(255, 235, 231);
+}
+
+.mid-article-head-content-1-a1 h6 {
+  font-size: 0.7rem;
+}
+
+.mid-article-head-content-1-body {
+  display: flex;
+  flex-direction: column;
+  line-height: 30px;
+  color: grey;
+  background-color: rgb(255, 250, 248);
+}
+
+.mid-article-head-content-1-body a:last-child {
+  border-top: 1px solid rgb(255, 205, 194);
+}
+
+.latest-deals-grid-header a:hover {
+  color: var(--color-black);
+}
+
+.mid-article-head-content-1 a:not(:last-child):hover {
+  color: var(--color-black);
+}
+
+.article-content-2-grid {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  margin-top: 5rem;
+}
+
+.article-content-2-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  width: 63rem;
+  grid-column-gap: 1.5rem;
+  grid-row-gap: 2.5rem;
+}
+
+.article-content-2 {
+  display: flex;
+  flex-direction: column;
+  cursor: pointer;
+}
+
+.article-content-2 h1 {
+  color: var(--color-orange);
+  font-size: 1.5rem;
+  margin-bottom: 0.7rem;
+}
+
+.article-content-2-pic {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  height: 16rem;
+  width: 100%;
+  color: var(--color-white);
+}
+
+.article-content-2-pic * {
+  margin-left: 0.4rem;
+  margin-bottom: 0.3rem;
+}
+
+.ac2p-1 {
+  background: linear-gradient(to bottom, rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)), url("./assets/article-1.png") no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.ac2p-2 {
+  background: linear-gradient(to bottom, rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)), url("./assets/article-2.jpg") no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.ac2p-3 {
+  background: linear-gradient(to bottom, rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)), url("./assets/article-3.png") no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.ac2p-4 {
+  background: linear-gradient(to bottom, rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)), url("./assets/article-4.jpg") no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.ac2p-5 {
+  background: linear-gradient(to bottom, rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)), url("./assets/article-5.jpg") no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.ac2p-6 {
+  background: linear-gradient(to bottom, rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)), url("./assets/article-6.jpg") no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.ac2p-7 {
+  background: linear-gradient(to bottom, rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)), url("./assets/article-7.jpg") no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.ac2p-8 {
```

Comment:
You are using the same couple of rules for several selectors. Instead, you should create only one class

Reply:
Implemented this